frontend/src/bundles/common/components/header/components/header-user-menu/header-user-menu.tsx

```diff
@@ -23,6 +24,7 @@ const HeaderUserMenu: React.FC<Properties> = () => {
 
     const handleSignOut = useCallback((): void => {
         void dispatch(authActions.signOut());
+        void dispatch(talentOnboardingActons.resetState());
```

Comment:
https://stackoverflow.com/a/61943631 we should reset all modules

Reply:
fixed